plugins/jitsi/jitsi_videobridge

```diff
@@ -50,7 +50,7 @@ case $1 in
     autoconf)
         if [ -x /usr/bin/curl ]; then
             if [ -x /usr/bin/jq ]; then
-	        curl -s -f -m 2 -I "$COLIBRI_URL" > /dev/null && echo "yes" && exit 0 || echo "no (no valid env.url)" && exit 0
+	        curl -s -f -m 2 -I "$COLIBRI_URL" | grep "Content-Type: application/json" > /dev/null && echo "yes" && exit 0 || echo "no (no valid env.url)" && exit 0
```

Comment:
That is a good approach!

Regarding `grep` and the check for existence: you can use `grep -q` without any redirects for this purpose.

btw: in the context of autoconf I would suggest to change the wording of *no valid env.url* to something like *invalid or empty response from colibri API ($COLIBRI_URL)*

Reply:
Good suggestion ... Thank you.